widget/textgrid.go

```diff
@@ -345,12 +369,15 @@ func (t *textGridRenderer) appendTextCell(str rune) {
 	text.TextStyle.Monospace = true
 
 	bg := canvas.NewRectangle(color.Transparent)
-	t.objects = append(t.objects, bg, text)
+
+	ul := canvas.NewLine(color.Transparent)
```

Comment:
Won't having 50% more objects always visible slow things down?
If adding them cannot be avoided at least avoid showing them... or maybe we need an optimisation in line drawing to ignore transparent lines? (I don't think that exists)...

Reply:
It doesn't appear any slower to me. I do wonder if we should have some benchmarks around this, however.

Reply:
even just running top should push the terminal and display CPU usage at the same time